src/main/java/org/openlmis/stockmanagement/repository/StockCardRepository.java

```diff
@@ -26,4 +58,16 @@ public StockCard getStockCardById(Long facilityId, Long id) {
   public List<StockCard> getStockCards(Long facilityId) {
     return mapper.getAllByFacility(facilityId);
   }
+
+  public void persistStockCardEntry(StockCardEntry entry) {
```

Comment:
Should we name this insertStockCardEntry instead of persist? Just so it would be more consistent.